NBAquotes/app/src/main/java/com/example/nbaquotes/BIBIQuote.java

```diff
@@ -0,0 +1,30 @@
+package com.example.nbaquotes;
+
+import android.util.Log;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class BIBIQuote {
+    public String mQuote;
+    public List<String> mQuotes = QuotesPull();
+
+    public BIBIQuote(){
+
+        int random = new Random().nextInt(7);
+        this.mQuote = this.mQuotes.get(random);
+    }
+
+    public List<String> QuotesPull() {
+        List<String> Quotes = new ArrayList<>();
```

Comment:
Think about a future requirement for a "title" of the quotes.
A class "Quote" could've been better than a list of strings. List<Quote>